File: Frontend/src/classess/HealthCenterStaff.ts
class HealthCenterStaff extends User {
  
  public constructor(name: string, mail: string, password: string, role: string, address: string, phoneNum: string, hesCode: string){
        super(name, mail, password, role, address, phoneNum, hesCode);
    }

  public closeForm(healthForm: HealthForm): boolean {
    //TODO
    return true;
  }
}

File: Backend/src/main/java/com/example/backend/HealthCenterStaff.java
package com.example.backend;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class HealthCenterStaff extends User{


    //Constructor
    public HealthCenterStaff(String name, String mail, String password, String role, String address, String phoneNum, String hesCode){
        super(name, mail, password, role, address, phoneNum, hesCode);
    }

    //Methods
    public boolean closeForm(HealthForm healthForm){
        // TO DO
        return true;
    }
}
